modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/generalized_plane_strain.md

```diff
@@ -139,6 +143,218 @@ N_{z}
+\boldsymbol{u}_x \\
+\epsilon_{yy} \\
+\boldsymbol{u}_z 
+\end{array}
+\right\}
+=
+\left\{
+\begin{array}{c}
+\boldsymbol{f}_x \\
+N_{y} \\
+\boldsymbol{f}_z 
+\end{array}
+\right\}
+\end{equation}
+
+The off-diagonal entries are nonzero, but not shown here.
+
+### For the generalized plane strain problem in the $y$-$z$ plane
+
+#### In-plane equilibrium equations
+
+The kinematical equations of the generalized plane strain problem are identical to those for the plane stress or strain problems, given as
+\begin{equation}
+\left\{\begin{matrix}
+\epsilon_{yy} \\
+\epsilon_{zz} \\
+\gamma_{yz}
+\end{matrix}\right\}
+=
+\left\{\begin{matrix}
+\frac{\partial v}{\partial y} \\
+\frac{\partial w}{\partial z} \\
+\frac{\partial w}{\partial y} + \frac{\partial v}{\partial z}
+\end{matrix}\right\}
+\end{equation}
+
+The equilibrium equations for the generalized plane strain problem in the are given as
+\begin{equation}
+\begin{matrix}
+\frac{\partial \sigma_{yy}}{\partial y} + \frac{\partial \sigma_{yz}}{\partial z} + \bar{f}_{y} = 0 \\
+\frac{\partial \sigma_{yz}}{\partial y} + \frac{\partial \sigma_{zz}}{\partial z} + \bar{f}_{z} = 0
+\end{matrix}
+\end{equation}
+in $A$ where $\left\{ \bar{f} \right\}$ are the body forces.
+
+The constitutive equations in terms of stress-strain relationship are given as
+\begin{equation}
+\left\{ \begin{matrix}
+\sigma_{xx} \\
+\sigma_{yy} \\
+\sigma_{zz} \\
+\tau_{xz}
+\end{matrix} \right\}
+=
+\left[ \begin{matrix}
+C_{11} & C_{12} & C_{13} & 0 \\
+C_{12} & C_{22} & C_{23} & 0 \\
+C_{13} & C_{23} & C_{33} & 0 \\
+0 & 0 & 0 & C_{44}
+\end{matrix} \right]
+\left( \left\{ \begin{matrix}
+\epsilon_{xx} \\
+\epsilon_{yy} \\
+\epsilon_{zz} \\
+\gamma_{yz}
+\end{matrix} \right\}
+-
+\left\{ \begin{matrix}
+\alpha_{x} \\
+\alpha_{y} \\
+\alpha_{z} \\
+0
+\end{matrix} \right\} \Delta T \right)
+\end{equation}
+where $C_{ij}$ are material's stiffness coefficients using Voigt notation, $\alpha_{x}$, $\alpha_{y}$ and $\alpha_{z}$ are the material's thermal expansion coefficients and $\Delta T$ is the change in temperature.
+
+#### Out-of-plane equilibrium equation
+
+A further condition is required associated with the out-of-plane direction. If a constant strain is prescribed as the deformation compatibility condition in the $x$-direction
+\begin{equation}
+\epsilon_{xx} = \bar{\epsilon}_{xx}
+\end{equation}
+Alternatively, a force as the stress resultant $\bar{N}_{x}$ in the $x$-direction can be prescribed, the condition is the equilibrium condition in $x$-direction, given as
+\begin{equation}
+N_{x} = \int_{A}{\sigma_{xx}dA} = \bar{N}_{x}
+\end{equation}
+The stress resultant $N_{x}$ conjugates with constant strain $\epsilon_{xx}$.
+
+The formulation above for generalized plane strain problem shares many similarities with that of the conventional plane stress or plane strain problem. +All the differences are associated with the introduction of an additional degree of freedom for the out-of-plane direction+.
+
```

Comment:
Can you pull this short paragraph out of the three out-of-plane sections and use it as a conclusion at the end in each case? If not, un-bold the last sentence so it is not as obvious that it is being repeated.